desc/profiles.py

```diff
@@ -89,21 +535,21 @@ class PowerSeriesProfile(Profile):
         coefficients of the series. If modes is not supplied, assumed to be in ascending order
         with no missing values. If modes is given, coefficients can be in any order or indexing.
     modes : array-like
-        mode numbers for the associated coefficients. eg a[modes[i]] = params[i]
+        mode numbers for the associated coefficients. eg a[modes[i]] = params[i].
     grid : Grid
         default grid to use for computing values using transform method
     sym : bool
         Whether the basis should only contain even powers (True) or all powers (False).
     name : str
-        name of the profile
+        name of the profile.
 
     """
 
     _io_attrs_ = Profile._io_attrs_ + ["_basis", "_transform"]
```

Comment:
On the `constrain-toroidal-current-density` branch, the `Configuration` class assumes that the `PowerSeriesProfile` class has a `sym` attribute. `profile.sym` gets set in the constructor but it is not included in the IO attributes, so it will not have this attribute if a profile is saved and then loaded. We should fix that here, either by adding it as an IO attribute or writing a getter method to determine the symmetry based on the modes.

Reply:
the symmetry is a property of the basis, which is saved. I'll add a property that just reads it from the basis